### PR TITLE
Ports Baystation wall and airlock changes [READY FOR TEST MERGE?]

### PR DESCRIPTION
**Pull request by NataKilar**
Need to do some save var testing on test server

Ports new Baystation sprites for walls and airlocks. Refactors airlocks into an overlay/paint based system, rather than having many different types of airlocks with single-use sprites. Walls are now paintable, as are airlocks. Alarms and APCs have been resprited to align with new wall sprites. Added low walls, which can have windows placed on them with rods+glass. Low walls are built in-hand with steel.

Mainly ported from here https://github.com/Baystation12/Baystation12/pull/21322, with various patches by the author and others ported as well.

### CODE REVIEW

File: code/game/machinery/doors/multi_tile.dm [hunk 1]
```diff
@@ -1,15 +1,25 @@
...
+	deny_file = 'icons/obj/doors/double/lights_deny.dmi'
+	lights_file = 'icons/obj/doors/double/lights_green.dmi'
+	panel_file = 'icons/obj/doors/double/panel.dmi'
+	welded_file = 'icons/obj/doors/double/welded.dmi'
+	emag_file = 'icons/obj/doors/double/emag.dmi'
 	width = 2
 	appearance_flags = 0
-/obj/machinery/door/airlock/multi_tile/should_save(var/datum/caller)
-	if(istype(caller, /turf))
-		var/turf/T = caller
```
Comment (Andrew-Fall) [inline code/game/machinery/doors/multi_tile.dm]: This should probably not have been removed

File: code/game/objects/structures/door_assembly.dm [hunk 2]
```diff
@@ -310,20 +227,25 @@
...
+			final_name = "Wired "
+			panel_overlay = image(panel_icon, "construction0")
 		if(2)
-			name = "Near Finished "
-	name += "[glass == 1 ? "Window " : ""][istext(glass) ? "[glass] Airlock" : base_name] Assembly"
-
-obj/structure/door_assembly/door_assembly_keyp
- 	base_icon_state = "keyp"
- 	base_name = "Keypad Airlock"
- 	glass = -1
```
Comment (Andrew-Fall) [inline code/game/objects/structures/door_assembly.dm]: This appears to be removing our keypad airlocks

File: code/game/turfs/simulated/walls.dm [hunk 3]
```diff
@@ -19,7 +19,15 @@
...
 	var/hitsound = 'sound/weapons/Genhit.ogg'
 	var/list/wall_connections = list("0", "0", "0", "0")
+	var/list/other_connections = list("0", "0", "0", "0")
 	var/floor_type = /turf/simulated/floor/plating //turf it leaves after destruction
+	var/paint_color
+	var/stripe_color
+	var/global/list/wall_stripe_cache = list()
+	var/list/blend_turfs = list(/turf/simulated/wall/cult)
+	var/list/blend_objects = list(/obj/machinery/door, /obj/structure/wall_frame, /obj/structure/grille, /obj/structure/window/reinforced/full, /obj/structure/window/reinforced/polarized/full, /obj/structure/window/shuttle, ,/obj/structure/window/phoronbasic/full, /obj/structure/window/phoronreinforced/full) // Objects which to blend with
+	var/list/noblend_objects = list(/obj/machinery/door/window) //Objects to avoid blending with (such as children of listed blend objects.
```
Comment (Andrew-Fall) [inline code/game/turfs/simulated/walls.dm]: Things like this should be constants or defines

**Review by Andrew-Fall — DISMISSED** [review]

File: code/game/turfs/simulated/walls.dm (hunk 3 above)
Comment (Andrew-Fall) [reply code/game/turfs/simulated/walls.dm]: You explained this can't be defines or const.

Comment (NataKilar) [reply code/game/turfs/simulated/walls.dm]: Fixed in latest commit

File: code/game/machinery/doors/multi_tile.dm (hunk 1 above)
Comment (NataKilar) [reply code/game/machinery/doors/multi_tile.dm]: Fixed in latest commit

**Review by Andrew-Fall — APPROVED** [review]
Looks fine, Awaiting Travis

**Pull request merged**